checkout/checkout-sdk-net
# CP-884: Adds the processing object to the PaymentRequest

### Jira link
 
-  [CP-884](https://checkout.atlassian.net/browse/CP-884)
 
### Functional comments
- Enable merchants to make payment requests using processing settings when using the .net sdk
 
### Technical comments
- I have modeled this as a `Dictionary<string, object>` as it is quite similar to `metadata`, but I'm happy to change this to something else if you think it is more suited. 
 
### How to test
- I have manually tested this with a local project; see the jira issue [CP-884](https://checkout.atlassian.net/browse/CP-884) for a code example.
 
### Other comments and links
I have a few questions;
1. Any recommendations on how I should unit test this?
2. Do I need to bump version numbers etc in this pull request?
 
[Code Review Guidelines](https://checkout.atlassian.net/wiki/spaces/CoreEng/pages/657064052/Code+Reviews)
